menuinst/platforms/linux.py

```diff
@@ -42,16 +42,18 @@ def __init__(self, *args, **kwargs):
                 os.environ.get("XDG_DATA_HOME", "~/.local/share")
             ).expanduser()
 
+        # The names have to be rendered often, so store them for convenience
+        self._rendered_name = self.render(self.name)
+        self._slugified_name = self.render(self.name, slug=True)
+
```

Comment:
Should we memoize `render` instead?

Reply:
The main problem I was trying to solve here is the human error component where rendering was forgotten. I could make this a cached property though if that makes more sense.

Reply:
I think folks would also forget to look for the private attribute. What if we make .name always rendered and offer a raw_name instead?

Reply:
I think that would just open up a different can of worms since we need to decide on when to render (and that may clash with #180).

I'm happy to remove the private class members and return the function calls (and even cache the rendering if that's better) if you think that's better.

Reply:
Ok, let's do that then. Use `render()` explicitly + lrucache the method.